src/lib.rs

```diff
@@ -1,8 +1,8 @@
 //! This crate contains the officially supported MongoDB Rust driver, a
 //! client side library that can be used to interact with MongoDB deployments
-//! in Rust applications. It uses the [`bson`](docs.rs/bson) crate for BSON support.
-//! The driver contains a fully async API that supports either [`tokio`](docs.rs/tokio) (default)
-//! or [`async-std`](docs.rs/async-std), depending on the feature flags set. The driver also has
+//! in Rust applications. It uses the [`bson`] crate for BSON support.
+//! The driver contains a fully async API that supports either [`tokio`] (default)
+//! or [`async-std`](https://docs.rs/async_std), depending on the feature flags set. The driver also has
```

Comment:
Using an absolute link here because async-std is an optional dependency, which causes docs tests to fail when the async-std feature is not enabled.